.travis.yml

```diff
@@ -26,7 +23,7 @@ addons:
 before_install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update ; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install gnu-tar dpkg libicns; fi
-  - gem install --no-rdoc fpm
+  - gem install --no-rdoc --no-ri fpm
```

Comment:
We cannot use modern option '--no-document' because we cannot use ruby 2.2 due to https://github.com/travis-ci/travis-ci/issues/4011